engine/src/main/java/io/camunda/zeebe/engine/api/Task.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.api;
+
+public interface Task {
+
+  TaskResult execute(TaskResultBuilder taskResultBuilder);
```

Comment:
:wrench: Maybe we add here also an comment that this is the intermediate solution and ideally the builder is in the end created in the ScheduleService consumer classes